### PR TITLE
Add software of my JRM paper

### PR DESCRIPTION
**Pull request by pazeshun**
### Modifications to existing code
- 95601e1: Enable to set `proba_threshold` of `fcn_object_segmentation.py` (see https://jsk-docs.readthedocs.io/projects/jsk_recognition/en/latest/jsk_perception/nodes/fcn_object_segmentation.html#parameters) in `object_segmentation_3d.launch`.

### CODE REVIEW

File: demos/baxtergv6_apc2016/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+roslaunch baxtergv6_apc2016 baxter.launch
+roslaunch baxtergv6_apc2016 setup_for_stow.launch
+roslaunch baxtergv6_apc2016 stow.launch json_dir:=$(rospack find baxtergv6_apc2016)/data/json/stow1
+```
+
+
+## Citation
+
+```
+@article{Hasegawa_2019jrm,
```
Comment (k-okada) [inline demos/baxtergv6_apc2016/README.md:34]: please add link to your publication https://www.fujipress.jp/jrm/rb/robot003100020289/ ?

Comment (pazeshun) [reply demos/baxtergv6_apc2016/README.md:36]: Thank you, I added the link

**Comment by pazeshun** [comment]
@knorth55 Could you review this?

**Comment by knorth55** [comment]
@pazeshun can i merge this?

**Comment by pazeshun** [comment]
Yes, please

**Comment by pazeshun** [comment]
Thank you very much

**Pull request merged**